tripal/src/Form/TripalEntityForm.php

```diff
@@ -196,33 +194,76 @@ public function form(array $form, FormStateInterface $form_state) {
+    $errors_found = FALSE;
+
+    // Retrieve the TripalEntityType (i.e. bundle).
     $bundle_id = $this->entity->getType();
-    $bundle_entity = \Drupal\tripal\Entity\TripalEntityType::load($bundle_id);
+    $bundle_entity = $this->entity->getBundle();
+
+    // First check the title format.
     $title_format = $bundle_entity->getTitleFormat();
-    $message = '';
+    $url = '/admin/structure/bio_data/manage/' . $bundle_id;
+    $title_message_suffix = ' We recommend you update the title format before creating any new content.'
+      . ' <a href=":url" target="_blank">Click here</a> to update the title format.';
     if (!preg_match('/\[.*\]/', $title_format)) {
-      $message = 'The Page Title Format for this content type does not contain any tokens.';
+      $message = $this->t('The Page Title Format for this content type does not contain any tokens. <strong>This will result in all titles for :bundle being the same!</strong>' . $title_message_suffix,
+        [':url' => $url,
+         ':bundle' => $bundle_entity->label()]
+      );
+      $this->messenger()->addError($message);
+      $errors_found = TRUE;
     }
     elseif ($title_format == 'Entity [TripalEntity__entity_id]') {
-      $message = 'The Page Title Format for this content type is the default generic format.';
-    }
-    if ($message) {
-      $url = '/admin/structure/bio_data/manage/' . $bundle_id;
-      $message .= ' You must update the title format before creating any new content.'
-        . ' <a href=":url">Click here</a> to update the title format.';
-      \Drupal::messenger()->addError($this->t($message, [':url' => $url]));
-      return FALSE;
+      $message = $this->t('The Page Title Format for this content type is the default generic format. <strong>This will result in very uninformative titles!</strong>' . $title_message_suffix);
+      $this->messenger()->addWarning($message);
     }
-    else {
-      return TRUE;
+
+    // Then check the URL Alias format.
+    $url_format = $bundle_entity->getURLFormat();
+    $url = '/admin/structure/bio_data/manage/' . $bundle_id;
+    $url_message_suffix = ' <a href=":url" target="_blank">Click here</a> to update the url format before creating any new content.';
+    if (!preg_match('/\[.*\]/', $url_format)) {
+      $message = $this->t(
+        'The URL Alias Format for this content type does not contain any tokens. <strong>This will result in no alias being added due to duplicate aliases.</strong>' . $url_message_suffix,
+        [':url' => $url]
+      );
+      $this->messenger()->addError($message);
+      $errors_found = TRUE;
+    } elseif ($url_format == '[TripalEntityType__term_label]/[TripalEntity__entity_id]') {
+      $message = $this->t('The URL Alias Format for this content type is the default generic format. We suggest updating it to include meaningful tokens for more readable URLs.' . $url_message_suffix,
+        [':url' => $url]
+      );
+      $this->messenger()->addWarning($message);
```

Comment:
I propose removing these lines as described in the previous comment

Reply:
Good point <3 I removed the URL alias warning in 07958be2b7bf9d81dea72a9a20a7dee49ad43e1a. Thanks!